samples/tab-deeplink/csharp/README.md

```diff
@@ -69,6 +69,9 @@ urlFragment: officedev-microsoft-teams-samples-tab-deeplink-csharp
     git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
     ```
    - Update channelID placeholer `<AddYourTeamsChannelId>` value in your `DeepLinkBot.cs` file with your particular channel id from any Team in Teams. (You can get it manually by clicking on 3 dots in any team's channel and fetch it's link and extract the channel id ) like `General` channel) Example: `19:cbe3683f25094106b826c9cada3afbe0@thread.skype`
+
+   - Navigate to `wwwroot/js/env.js` file and update your AppId at placeholder `<<App-ID>>` (You can get it manually frrm [teams admin portal](https://admin.teams.microsoft.com/).
+	- ![personal-AddedBot ](DeepLinkBot/images/TeamsAdminPortal.png)
    
    - Update the `appsettings.json` configuration for the bot to use the MicrosoftAppId, MicrosoftAppPassword, BaseURL ,
```

Comment:
Update Readme file for descriptions and capabilities.

Reply:
updated readme